test/features/comments.feature

```diff
@@ -79,29 +79,102 @@ Feature: Comments
     Scenario: When a logged-in user submits a reply comment on a Dataset, the comment should display within 10 seconds
         Given "CKANUser" as the persona
         When I log in
-        Then I go to dataset "warandpeace"
+        Then I go to dataset "warandpeace" comments
         Then I submit a reply with comment "This is a reply"
         Then I take a screenshot
         Then I should see "This is a reply" within 10 seconds
 
     @comment-delete
-    Scenario: When an Org Admin visits a dataset belonging to their organisation, they can delete a comment and the comment will be marked as deleted.
+    Scenario: When an Org Admin visits a dataset belonging to their organisation, they can delete a comment and should not see text "This comment was deleted." 
         Given "TestOrgAdmin" as the persona
         When I log in
-        Then I go to dataset "warandpeace"
+        Then I go to dataset "warandpeace" comments
+        And I press the element with xpath "//a[@title='Delete comment']"
+        Then I should see "Are you sure you want to delete this comment?" within 1 seconds
+        Then I press the element with xpath "//button[contains(string(), 'Confirm')]"
+        Then I take a screenshot
+        Then I should not see "This comment was deleted." within 2 seconds
+
+    @comment-delete
+    Scenario: When an Org Admin visits a data request belonging to their organisation, they can delete a comment and should not see text "This comment was deleted." 
+        Given "Admin" as the persona
+        When I log in
+        And I go to data request "Test Request" comments
+        And I press the element with xpath "//a[@title='Delete comment']"
+        Then I should see "Are you sure you want to delete this comment?" within 1 seconds
+        Then I press the element with xpath "//button[contains(string(), 'Confirm')]"
+        Then I take a screenshot
+        Then I should not see "This comment was deleted." within 2 seconds
+
+    @comment-add
+    Scenario: When a logged-in user submits a comment on a Dataset the page should be scrolled to the highlighted comment I just posted.
```

Comment:
This sentence switches between third-person and first-person.

Reply:
@ThrawnCA have addressed via https://github.com/MarkCalvert/ckanext-ytp-comments/commit/036112d3f95d8b4f99b9471c3c6c0d42903dd04c